client/src/components/Upload/UploadModal.vue

```diff
@@ -331,6 +361,35 @@ const UploadModal = {
                 }
                 data.payload.inputs = JSON.stringify(inputs);
             }
+
+            return data;
+        },
+        toFetchData: function (items, history_id) {
+            var data = {
+                history_id: history_id,
+                targets: [
+                    {
+                        destination: { type: "hdas" },
+                        elements: [],
+                        name: "",
+                    },
+                ],
+                auto_decompress: true,
+            };
+
+            const urls = items[0].get("url_paste").split("\n");
+            for (var index in urls) {
+                var url = urls[index].trim();
+                if (url != "") {
+                    var element = {
+                        url: urls[index].trim(),
+                        src: "url",
+                        dbkey: "?",
```

Comment:
dbkey and ext should be coming from the form in some way I think, these make sense as defaults but if they're overridden that value should be used I think.

Reply:
```
                        inputs[`${prefix}space_to_tab`] = (it.get("space_to_tab") && "Yes") || null;
                        inputs[`${prefix}to_posix_lines`] = (it.get("to_posix_lines") && "Yes") || null;
                        inputs[`${prefix}dbkey`] = it.get("genome", null);
                        inputs[`${prefix}file_type`] = it.get("extension", null);
```